Gemfile.lock

```diff
@@ -418,6 +430,11 @@ GEM
       simplecov-html (~> 0.10.0)
     simplecov-html (0.10.0)
     single_cov (0.2.2)
+    slack-api (1.2.3)
```

Comment:
:heart:

On Sun, Jul 17, 2016 at 3:17 PM, Ben Straub <notifications@github.com>
wrote:

> In Gemfile.lock
> <https://github.com/zendesk/samson/pull/1130#discussion_r71090824>:
>
> > @@ -418,6 +430,11 @@ GEM
> >        simplecov-html (~> 0.10.0)
> >      simplecov-html (0.10.0)
> >      single_cov (0.2.2)
> > +    slack-api (1.2.3)
>
> Good point. Removed.
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1130/files/d008e855296f051e83d429a560e4e921b94fd917#r71090824>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZ-XeQwiCJ6Vq46AjVgdW2rv_3AK8ks5qWqn0gaJpZM4JIRHk>
> .
>